imports/ui/components/dashboard/match_card.jsx

```diff
@@ -1,33 +1,38 @@
 import React, { Component } from 'react'
-import { Card, Dropdown, ButtonGroup, Button } from 'react-bootstrap'
+import { Card, CardDeck, Dropdown, ButtonGroup, Button } from 'react-bootstrap'
+import '/imports/ui/styles/_cards.scss';
 
 class MatchCard extends Component {
   constructor (props) {
     super(props)
+    //console.log(props)
   }
 
   render () {
     return (
-      <Card style={{ width: '33%' }}>
+      <Card>
         <Card.Body>
-          <Card.Title className='font-weight-normal text-capitalize'>
-            { this.props.name }
+          <Card.Title className='font-weight-normal'>
+          { this.props.oneLineIntro }
```

Comment:
I started deconstructing props at the beginning of a `render()` method as I thought it's made the JSX more readable. Any thoughts on establishing that as a pattern?

```jsx
render () {
   const { oneLineIntro, lookingFor } = this.props

   return (
      <Card>
          <Card.Body>
              <Card.Title className='font-weight-normal'> { oneLineIntro } </Card.Title>
              <Card.Text> { lookingFor } </Card.Text>
          ...
   )
}

Reply:
++ Thanks, I've made this change.